tests/unit/render/TerraformRender.spec.js

```diff
@@ -0,0 +1,119 @@
+import TerraformRender from 'src/render/TerraformRender';
+import TerraformFile from 'src/models/TerraformFile';
+import TerraformResource from 'src/models/TerraformResource';
+import TerraformData from 'src/models/TerraformData';
+import TerraformVariable from 'src/models/TerraformVariable';
+import TerraformModule from 'src/models/TerraformModule';
+import TerraformProvider from 'src/models/TerraformProvider';
+import TerraformAttribute from 'src/models/TerraformAttribute';
+
+describe('Test TerraformRender', () => {
+  const render = new TerraformRender();
+  describe('Test method: render', () => {
+    it('With databaseExemple.tf', () => {
+      console.log(render.render(
```

Comment:
It's not a test, please load the file ".tf" and verify that file content is equal to TerraformRender.render